### PR TITLE
Feature/hotspot edit mode

### PR DESCRIPTION
**Pull request by debreb**

### CODE REVIEW

**Comment by debreb** [comment]
https://user-images.githubusercontent.com/70328824/158178080-f1bae6c9-c529-4a5d-8291-3e03b2edf359.mp4

File: screens/AddHotspotScreen.tsx [hunk 1]
```diff
@@ -187,6 +207,38 @@ const getStyles = (theme: ReactNativePaper.Theme) =>
...
       fontFamily: 'Nunito_400Regular',
     },
+    imageView: {
+      height: scaledHeight + 28,
+      marginTop: 32,
+      width: '100%',
+    },
+    image: {
+      width: '100%',
+      maxHeight: scaledHeight,
```
Comment (mveres) [inline screens/AddHotspotScreen.tsx:217]: same here, are we sure it's needed?

File: screens/AddHotspotScreen.tsx [hunk 2]
```diff
@@ -187,6 +207,38 @@ const getStyles = (theme: ReactNativePaper.Theme) =>
...
+      position: 'absolute',
```
Comment (mveres) [inline screens/AddHotspotScreen.tsx:218]: can't we avoid absolute positioning here?

**Review by mveres — CHANGES_REQUESTED** [review]

File: screens/AddHotspotScreen.tsx [hunk 3]
```diff
@@ -187,6 +207,38 @@ const getStyles = (theme: ReactNativePaper.Theme) =>
       fontSize: 15,
       fontFamily: 'Nunito_400Regular',
     },
+    imageView: {
+      height: scaledHeight + 28,
```
[the remark this answers is not in the document]
Comment (debreb) [reply screens/AddHotspotScreen.tsx:211]: I'm okay with this solution, but in this case we need to ask our designer to resize some of the images,
for ex.: 
Screenshot 2022-03-15 at 19 17 21

**Review by mveres — APPROVED** [review]

**Pull request merged**